book/schedule.yaml

```diff
@@ -69,19 +70,20 @@ tabs:
     time: 8:30 - 9:15
     leads:
     - Karina Zikan
+    - Zach Fair
```

Comment:
This should be 'Zachary Fair' so the splash page links his name with his profile (automatically)